### PR TITLE
Improve ComponentRequester tests

### LINKED ISSUE #955: Improve unit test in ComponentRequester class
## Use case

ComponentRequester test class is making use of any() in some scenarios that can cause a false positive test.

## Proposal

Replace this cases for specific mocks to reproduce in a more reliable way the correct flow

### PR DESCRIPTION
**Pull request by gabrielleonardozup**
### Related Issues

Closes #955
### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

File: backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/CustomAndroidAction.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.sample.builder
+
+import br.com.zup.beagle.annotation.RegisterAction
+import br.com.zup.beagle.widget.action.Action
+
+@RegisterAction
+class CustomAndroidAction: Action {
```
Comment (botbeaglezup) [inline backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/CustomAndroidAction.kt:23]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/f4b2ad178c18293291ff2ca667c38e5045f49596/backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/CustomAndroidAction.kt#L23"></span>The class or object CustomAndroidAction is empty.</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/sample/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -19,4 +19,6 @@
     <string name="tv_screen_deeplink">Screen DeepLink</string>
     <string name="btn_title_come_back">come back</string>
```
Comment (botbeaglezup) [inline android/sample/src/main/res/values/strings.xml:20]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/f4b2ad178c18293291ff2ca667c38e5045f49596/android/sample/src/main/res/values/strings.xml#L20"></span>The resource <code>R.string.btn_title_come_back</code> appears to be unused</td>
    </tr>
  </tbody>
</table>

File: android/sample/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -19,4 +19,6 @@
...
     <string name="title_btn_retry">Try again</string>
```
Comment (botbeaglezup) [inline android/sample/src/main/res/values/strings.xml:21]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/f4b2ad178c18293291ff2ca667c38e5045f49596/android/sample/src/main/res/values/strings.xml#L21"></span>The resource <code>R.string.title_btn_retry</code> appears to be unused</td>
    </tr>
  </tbody>
</table>

File: android/sample/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -19,4 +19,6 @@
...
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (botbeaglezup) [inline android/sample/src/main/res/values/strings.xml:23]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/f4b2ad178c18293291ff2ca667c38e5045f49596/android/sample/src/main/res/values/strings.xml#L23"></span>The resource <code>R.string.hello_blank_fragment</code> appears to be unused</td>
    </tr>
  </tbody>
</table>

**Comment by gabrielleonardozup** [comment]
This was close because there was so many wrong files updated

**Pull request closed without merging**